worker/uniter/runner/runner.go

```diff
@@ -40,10 +40,21 @@ var logger = loggo.GetLogger("juju.worker.uniter.runner")
 type runMode int
 
 const (
-	runOnLocal runMode = iota
+	runOnUnknown runMode = iota
+	runOnLocal
 	runOnRemote
 )
 
+// RunLocation dictates where to execute commands.
+type RunLocation string
+
+const (
+	// Uniter runs where the uniter is running.
+	Uniter = RunLocation("uniter")
+	// Unit runs where the unit/workload is running.
+	Unit = RunLocation("unit")
```

Comment:
Would prefer operator and workload internally to to make things consistent.
k8s doesn't really have uniters as top level entities - the artefacts are charm operator pod and workload pod; these 2 pods are what the exec target is